pkg/kubectl/resource_printer.go

```diff
@@ -479,6 +482,40 @@ func printPodTemplateList(podList *api.PodTemplateList, w io.Writer) error {
 	return nil
 }
 
+func printJob(job *api.Job, w io.Writer) error {
```

Comment:
Please display events related to the Job similar to what we now do for ReplicationController